### ISSUE TITLE
Colorpicker widget for css textarea

### ISSUE THREAD

**Issue by indigoxela** [backdrop-contrib/pelerine#7]
@stpaultim another idea: what about adding a tiny colorpicker to the css edit form item?

This might make selecting colors easier. If you're curious - a new [experimental branch](https://github.com/backdrop-contrib/pelerine/tree/exp-colorpicker-widget) is available.

I'm not sure yet if the user experience is better with that widget, especially regarding the automatic insert in the textarea. Testing and feedback is welcome.

Here's a screenshot:

![screenshot-colorpick](https://user-images.githubusercontent.com/842066/89866140-529cf800-dbaf-11ea-9e77-0ac71452ba6a.png)

**Comment by stpaultim** [comment]
@indigoxela - I love this experimenting that you are doing. Here is my quick and direct feedback on this feature. 

1) It certainly feels like it could be useful to some users. Personally, I have a color browser widget added to my browser already, so it's pretty easy for me to get color values. This is a tiny bit easier, but not much. Having said that, I am not the average site builder for whom finding color values is probably a bit more work. 

2) I love that you are testing these features in Pelerine. However, I wonder if Pelerine is the right theme for these features. One of the appeals of Pelerine is the single click, choose your background setting. Nothing more, very simple. I suspect that users looking for this kind of simple select your own background theme, may not be looking to add custom css and needing to find color values. You have already disabled the color module for this theme. 

3) If you are going to add these kinds of features for Pelerine, why not also have a limited choice implementation of the color module (for the options that make sense)?

4) I do think that someone using a theme like Tatsu (my contrib theme targeted at site builders) might really appreciate these tools. I will probably steal this code and use it in Tatsu! THANK YOU!

5) I haven't looked at Lateral as closely as Pelerine, but it might be a better fit for these features. 

6) If you are going to put this in Pelerine, maybe put it in a collapsible field set of "advanced" features? I think I will do this in Tatsu.

This is my idea for Tatsu, but I will have the Advanced Field Set collapsed by default. You will only see this feature if you open the advanced fieldset, which is a kind of warning that users should be careful with it.

![Tatsu_settings___Simplo](https://user-images.githubusercontent.com/3144571/89935760-5002c880-dbd8-11ea-8a3d-aec748099392.png)

These are some initial thoughts. None of this is meant to be a strong opinion on whether or not you should keep this in Pelerine. I certainly don't think it's a bad thing, I am just not sure how useful it will be to users of this specific theme, unless you go ahead and add more feature like this.

**Comment by indigoxela** [comment]
> One of the appeals of Pelerine is the single click

Yes that's what I had in mind when creating this theme - keep it a simple as possible.

> You have already disabled the color module for this theme

The color module - while a cool feature - isn't always easy to handle. Sometimes it's a frustrating time-eating monster for users. And a challenge for developers.
Lateral uses the color module btw.

This small javascript widget has a different purpose - it doesn't create stylesheets on its own, but helps selecting a color and inputs the value in the textarea.

> I will probably steal this code and use it in Tatsu!

Sure, you're welcome. - It's not "stealing", actually... :wink: 

> ...I am just not sure how useful it will be to users of this specific theme

Neither am I, that's why I asked for feedback. And I appreciate yours.
